Automattic/wp-calypso
# My Home: Handle view skipping server-side

Follows up #42239.
Requires D43508-code.

#### Changes proposed in this Pull Request

This PR uses the new endpoint exposed in D43508-code for skipping the current view and moving to the next one, effectively allowing us to remove the work-around for handling the race conditions we added in https://github.com/Automattic/wp-calypso/pull/42239/commits/cc55ebbf888ea21b8b713e31a3cf5bc7a0cf061a.

#### Testing instructions

Create a new site and make sure the next applicable view is displayed when:
- Clicked on "Get started" in the site created notice.
- Clicked on "Show site setup" in the site launch notice (when the site setup is still incomplete).
- Clicked on "Show me what's next" in the site launch notice (if site was launched as last step during site setup).
- Clicked on "Show me what's next" in the site setup complete notice.
- Clicked on "Show me what's next" in the site migrated notice.
